.github/workflows/sanity_checks.yml

```diff
@@ -39,6 +39,11 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Create non privileged user
+        run: |
+          adduser wrapdb --disabled-password
+          su wrapdb
```

Comment:
AFAIK this step runs in its own process so once it's completed it will exit and next step ("Sanity Checks") will run in a fresh process. So it won't work (indeed it doesn't).

I'd try doing this in "Sanity Checks" step below:

```yaml
su - wrapdb -c "./tools/sanity_checks.py"
```

However it won't work because it won't find `./tools/sanity_checks.py`. It should work if we add absolute path (not sure which one it is).